homeassistant/components/scrape/sensor.py

```diff
@@ -179,11 +218,23 @@ def _async_update_from_rest_data(self) -> None:
             SensorDeviceClass.TIMESTAMP,
         }:
             self._attr_native_value = value
+            _LOGGER.debug("_attr_native_value=%s", self._attr_native_value)
+            _LOGGER.debug("raw_value=%s", raw_value)
+            self._process_manual_data(raw_value)
             return
 
         self._attr_native_value = async_parse_date_datetime(
             value, self.entity_id, self.device_class
         )
+        self._process_manual_data(raw_value)
+        self.async_write_ha_state()
+
+    @property
+    def available(self) -> bool:
+        """Return if entity is available."""
+        available1 = CoordinatorEntity.available.fget(self)  # type: ignore[attr-defined]
+        available2 = ManualTriggerEntity.available.fget(self)  # type: ignore[attr-defined]
+        return bool(available1 and available2)
```

Comment:
That was already the case as the coordinator was already implemented e.g. no fetching information then it would go unavailable.
This now adds the ability for the user to provide a template to control `available` also.